.meta/transforms/transaction.toml

```diff
@@ -0,0 +1,248 @@
+[transforms.transaction]
+title = "Transaction"
+allow_you_to_description = "combines events of a matching transaction into a single event"
+beta = true
+common = false
+function_category = "aggregate"
+input_types = ["log"]
+output_types = ["log"]
+requirements = {}
+
+<%= render("_partials/fields/_component_options.toml", type: "transform", name: "transaction") %>
+
+[transforms.transaction.options.expire_after_ms]
+type = "int"
+common = false
+required = false
+default = 30000
+description = """\
+A maximum period of time to wait before a transaction should be considered \
+complete after not having received another event.\
+"""
+
+[transforms.transaction.options.flush_period_ms]
+type = "int"
+common = false
+required = false
+default = 1000
+description = """\
+Controls the frequency that Vector checks for (and flushes) expired \
+transactions.\
+"""
+
+[transforms.transaction.options.identifier_fields]
+type = "[string]"
+common = true
+required = false
+default = []
+examples = [["request_id"],["user_id","transaction_id"]]
+description = """\
+An ordered list of fields to distinguish transactions by. Each transaction has \
+a separate event combining state, and these fields should be used to prevent
+events from unrelated transactions from combining.\
+"""
+
+[transforms.transaction.options.merge_strategies]
+type = "table"
+common = false
+required = false
+description = """\
+A map of field names to custom merge strategies. For each field specified this \
```

Comment:
Which one of the given enums is the default behavior? `sum`?

Reply:
It's type-specific, as detailed below.